src/main/java/de/unijena/cheminf/clustering/desccalc/Descriptor.java

```diff
@@ -98,21 +98,26 @@
 import java.util.Arrays;
 import java.util.EnumMap;
 import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.IntStream;
 
 /**
  * Descriptor related calculations based on the CDK for the enrichment of data vectors.
- * Note: There are 3 "public static boolean setDescriptorsForMolecules...()" methods with different forms of
- * (parallelized) calculation since individual CDK descriptor calculation classes are unfortunately NOT thread-safe.
+ * Note: There are 4 "public static boolean setDescriptorsForMolecules...()" methods with different forms of
+ * (parallelized) calculation. There is single molecule processing or batch processing for large datastructures.
+ * Single and Batch processing can be used with either an IAtomContainer array or with a String Array of SMILES codes.
  * Note: For adding a new descriptor, go to "Add new descriptor information here!"
  *
```

Comment:
add `<p>` in the empty line, so that it is represented correctly in the formatted JavaDoc.